lib/task/import/csvImportTask.class.php

```diff
@@ -482,18 +482,15 @@ public function execute($arguments = [], $options = [])
                         } elseif (null !== QubitInformationObject::getById($self->rowStatusVars['parentId'])) {
                             $parentId = $self->rowStatusVars['parentId'];
                         } else {
+                            $parentId = $self->status['defaultParentId'];
```

Comment:
In the original code there is this conditional:

```
        if ($notImportingTranslation) {
            $self->object->parentId = QubitInformationObject::ROOT_ID;
        }
```

Is this no longer necessary?

Reply:
Yeah, it's redundant as the same check gets done a little later in the code.

Reply:
Ah okay. 👍